includes/Admin/Admin_Page.php

```diff
@@ -150,16 +150,23 @@ private function get_available_plugins() {
 	 * Renders the "Plugin Check" page.
 	 *
 	 * @since n.e.x.t
+	 *
+	 * @global array  $available_plugins        The list of available plugins.
+	 * @global string $selected_plugin_basename The selected plugin basename.
+	 * @global array  $categories               An array of categories.
+	 * @global mixed  $user_settings            The user interface setting value.
 	 */
 	public function render_page() {
-		global $available_plugins, $selected_plugin_basename, $categories;
+		global $available_plugins, $selected_plugin_basename, $categories, $user_settings;
 
 		$available_plugins = $this->get_available_plugins();
 
 		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
 
 		$categories = Check_Categories::get_categories();
 
+		$user_settings = get_user_setting( 'category_preferences' );
```

Comment:
Let's handle the "exploding" into an array here, rather than in every `foreach` loop in the template.

Reply:
Thanks for raising, Nice catch. Addressed in https://github.com/WordPress/plugin-check/pull/304/commits/678e5341421ec207b56bd1c6c2cd4db93a3fb157